### PR TITLE
feat: CQDG-32 implement study page

### PR DESCRIPTION
**Pull request by atoulous**
# FEAT CQDG-32

IMPLEMENT STUDY ENTITY PAGE

- closes #TICKET_NUMBER

## Description

[[JIRA LINK]](https://ferlab-crsj.atlassian.net/browse/CQDG-32)

Acceptance Criterias

## Validation

- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

## Screenshot
image

image
image

## QA

Steps to validate
Url (storybook, ...)
...

## Mention

### CODE REVIEW

**Review by adipaul1981 — APPROVED** [review]

File: src/components/reports/DownloadClinicalDataDropdown.tsx [hunk 1]
```diff
@@ -47,7 +51,7 @@ const DownloadClinicalDataDropdown = ({ participantIds, sqon, type = 'default' }
   return (
     <Dropdown
       key="actionDropdown"
-      disabled={participantIds.length === 0}
+      // disabled={participantIds.length === 0}
```
Comment (ethienneroy) [inline src/components/reports/DownloadClinicalDataDropdown.tsx]: à garder ?

File: src/views/StudyEntity/index.tsx [hunk 2]
```diff
@@ -53,7 +50,7 @@ const StudyEntity = () => {
     SUMMARY = 'summary',
     STATISTIC = 'statistic',
     DATA_ACCESS = 'data_access',
-    DATA_SET = 'data_set',
+    // DATA_SET = 'data_set',
```
Comment (ethienneroy) [inline src/views/StudyEntity/index.tsx]: à garder ?

File: src/components/reports/DownloadClinicalDataDropdown.tsx (hunk 1 above)
Comment (atoulous) [reply src/components/reports/DownloadClinicalDataDropdown.tsx]: Nop

File: src/views/StudyEntity/index.tsx (hunk 2 above)
Comment (atoulous) [reply src/views/StudyEntity/index.tsx]: Oui mais je peux virer

**Review by kstonge — APPROVED** [review]

**Pull request merged**